client/src/features/catalog/Catalog.tsx

```diff
@@ -1,6 +1,8 @@
+import { Grid } from "@mui/material";
 import { useState, useEffect } from "react";
 import { Product } from "../../app/models/product";
 import ProductList from "./ProductList";
+import '../../css/Products.css';
```

Comment:
use either double quotes either single quotes in the markup, code should be consistent. Do not use different quotes in the similar places of code.